tests/TestCase/Console/CommandCollectionTest.php

```diff
@@ -238,13 +238,11 @@ public function testAutoDiscoverApp()
 
         $this->assertTrue($collection->has('app'));
         $this->assertTrue($collection->has('demo'));
-        $this->assertTrue($collection->has('i18m'));
         $this->assertTrue($collection->has('sample'));
         $this->assertTrue($collection->has('testing_dispatch'));
 
         $this->assertSame('TestApp\Shell\AppShell', $collection->get('app'));
         $this->assertSame('TestApp\Command\DemoCommand', $collection->get('demo'));
-        $this->assertSame('TestApp\Shell\I18mShell', $collection->get('i18m'));
```

Comment:
Where did this command/shell go?

Reply:
It was lost due to a bad merge by me, restored again.